From rdunlap@xenotime.net Mon Aug  7 15:56:22 2006
Date: Mon, 7 Aug 2006 15:56:40 -0700
From: "Randy.Dunlap" <rdunlap@xenotime.net>
To: lkml <linux-kernel@vger.kernel.org>
Cc: akpm <akpm@osdl.org>, gregkh <greg@kroah.com>
Subject: usbnet: printk format warning
Message-Id: <20060807155640.63e59e6b.rdunlap@xenotime.net>

From: Randy Dunlap <rdunlap@xenotime.net>

Fix printk format warning(s):
drivers/usb/net/usbnet.c:654: warning: int format, different type arg (arg 3)

Can't say that I understand this one...

Signed-off-by: Randy Dunlap <rdunlap@xenotime.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/net/usbnet.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/net/usbnet.c
+++ gregkh-2.6/drivers/usb/net/usbnet.c
@@ -652,7 +652,7 @@ static int usbnet_open (struct net_devic
 			framing = "simple";
 
 		devinfo (dev, "open: enable queueing "
-				"(rx %d, tx %d) mtu %d %s framing",
+				"(rx %ld, tx %d) mtu %u %s framing",
 			RX_QLEN (dev), TX_QLEN (dev), dev->net->mtu,
 			framing);
 	}
